src/common/Cryptography/OpenSSLCrypto.h

```diff
@@ -19,17 +19,25 @@
 #define OPENSSL_CRYPTO_H
 
 #include "Define.h"
+#include <openssl/opensslv.h>
 
 /**
 * A group of functions which setup openssl crypto module to work properly in multithreaded enviroment
 * If not setup properly - it will crash
 */
 namespace OpenSSLCrypto
 {
+
+#if defined(OPENSSL_VERSION_NUMBER) && OPENSSL_VERSION_NUMBER < 0x1010000fL
     /// Needs to be called before threads using openssl are spawned
     TC_COMMON_API void threadsSetup();
     /// Needs to be called after threads using openssl are despawned
     TC_COMMON_API void threadsCleanup();
+#else
+    TC_COMMON_API void threadsSetup() { };
```

Comment:
`TC_COMMON_API` not needed here, it's only needed if the definition is in cpp

Reply:
This is also true. Fixed.